modules/integration/tests-ui-integration/tests-ui/src/test/java/org/wso2/ds/ui/integration/test/dashboard/CreateCustomDashboardPerUser.java

```diff
@@ -182,8 +179,9 @@ public void testCustomizeButtonDashboard() throws Exception {
         DSWebDriver driver = getDriver();
 
         driver.findElement(By.cssSelector("a.ues-copy")).click();
+        driver.findElement(By.cssSelector("a.ues-pages-toggle")).click();
+        driver.findElement(By.cssSelector("#ues-dashboard-pages .ues-page-item.active .accordion-toggle")).click();
```

Comment:
These classes are needed to identify the element since there are many accordion-toggles.